components/constant_contact/actions/update-contact/update-contact.mjs

```diff
@@ -14,6 +14,7 @@ export default {
         constantContact,
         "contactId",
       ],
+      reloadProps: true,
```

Comment:
Is `reloadProps` needed here? I don't see `contactId` being used in `additionalProps()`.

Reply:
It's the same of the emailAddress from add-contact